bin/templates/project/app/build.gradle

```diff
@@ -322,3 +323,7 @@ for (def func : cdvPluginPostBuildExtras) {
 if (hasProperty('postBuildExtras')) {
     postBuildExtras()
 }
+
+if (cdvHelpers.getConfigPreference('EnableGoogleServicesPlugin', 'false').toBoolean()) {
+    apply plugin: 'com.google.gms.google-services'
```

Comment:
@chemerisuk I've had to make this line:

```
apply plugin: 'com.google.gms.googleservices.GoogleServicesPlugin'
```

because the class must be used instead of id(string) to be able to apply plugin from non-root gradle file. I haven't tested the PR yet but I'm assuming the reason you are using the ID is because it is a root gradle file?

Reply:
Yes, because this is the root Gradle file. Worked fine on my side.

For testing I commented changes from cordova-support-google-services and inserted PR code manually. Then rebuilt the project and tried to start (btw google services v15 trigger error at runtime).

Reply:
👍